app/views/NewMessageView.tsx

```diff
@@ -84,6 +85,7 @@ class NewMessageView extends React.Component<INewMessageViewProps, INewMessageVi
 		this.init();
 		this.state = {
 			search: [],
+			searchText: '',
```

Comment:
I see we had to this kind of state on all places, but what if you lift it to Searchbox instead?
The component would be able to control is own state and theses changes wouldn't be needed.

Something like this:
```jsx
const SearchBox = ({ onChangeText, onSubmitEditing, testID }: ISearchBox): JSX.Element => {
	const { colors, theme } = useTheme();
	const [text, setText] = useState('');

	const internalOnChangeText = useCallback(value => {
		setText(value);
		onChangeText?.(value);
	}, []);

	return (
		<View testID='searchbox'>
			<FormTextInput
				autoCapitalize='none'
				autoCorrect={false}
				blurOnSubmit
				placeholder={I18n.t('Search')}
				placeholderTextColor={colors.auxiliaryTintColor}
				returnKeyType='search'
				underlineColorAndroid='transparent'
				containerStyle={{ margin: 16 }}
				onSubmitEditing={onSubmitEditing}
				value={text}
				theme={theme}
				testID={testID}
				onChangeText={internalOnChangeText}
				onCancelSearch={() => internalOnChangeText('')}
				iconRight={'search'}
			/>
		</View>
	);
};
```